packages/apps/hr/src/hooks/use-hr-header-actions.js

```diff
@@ -74,10 +77,32 @@ const useTimerListAction = () => {
           iconName: 'plus-lg',
           title: I18n.t('Hr_NewTimer'),
           iconColor: Colors.primaryColor.background,
-          onPress: () => console.log('Header button pressed.'),
+          onPress: () => navigation.navigate('ActiveTimerFormScreen'),
```

Comment:
You also need to add access to this screen on update button of TimeCard for timer list screen